editor/import_defaults_editor.cpp

```diff
@@ -187,11 +188,15 @@ void ImportDefaultsEditor::clear() {
 	}
 	names.sort();
 
+	if (is_first && names.size() > 0)
+		last_selected = names[0];
+
```

Comment:
`is_first` is not needed.

```suggestion
	if (last_selected.is_empty() && !names.is_empty()) {
		last_selected = names[0];
	}
```

Reply:
Here's what I thought at first, have a bool here can make code more understandable. But you are right, I will add a comment to the code.